lambdas/es/indexer/requirements.txt

```diff
@@ -17,10 +17,10 @@ jupyter-core==4.11.2
 nbformat==4.4.0
 numpy==1.21.5
 openpyxl==3.0.7
-pandas==1.1.5
+pandas==1.3.5
 pdfminer.six==20201018
 psutil==5.7.0
-pyarrow==2.0.0
+pyarrow==12.0.0
```

Comment:
That's quite a version bump. Should probably check if there are any breaking changes?

Reply:
Yeah, I'll check their release notes before merging. The sucky thing is local dev isn't possible without this huge bump. I can put it in a separate PR if you prefer.